src/app/project/project-details-tab/project-details-tab.component.ts

```diff
@@ -160,10 +160,10 @@ export class ProjectDetailsTabComponent implements OnInit, AfterViewInit, OnDest
 
     // draw project marker
     if (this.shapefile[0] && this.shapefile[0]._id && this.shapefile[0].documentFileName && this.shapefile[0].documentFileName.length > 0) {
-      console.log('Shapefile', this.shapefile);
+      const shapeFileStyle = { color: this.project.shapeFileColour || 'rgb(51, 136, 255)' };
```

Comment:
`rgb(51, 136, 255)` could maybe be defined in a constant somewhere so that it's clear what it means (`DEFAULT_SHAPEFILE_COLOUR` or something).

Reply:
No worries, it's been saved with the other constants in the app.